docs/USER_GUIDE.adoc

```diff
@@ -23,6 +22,14 @@ as a multi-branch project source in two different ways:
 
  IMPORTANT: This plugin is not compatible with versions of Bitbucket Server previous to 4.0.
 
+[id=bitbucket-server-7]
+== BitBucket Server 7 compatibility
+
+_BitBucket Server 7.x_ is supported but does no longer support lightweight checkout for pull requests.
```

Comment:
```suggestion
_BitBucket Server 7.x_ is supported but does no longer support lightweight checkout for pull requests when merge strategy is used for build.
```

Lightweight checkout works if we don't use merge build strategy.

Reply:
Ah I said that correct on the first part but it is missing here indeed. Good catch!